compute_sdk/globus_compute_sdk/sdk/client.py

```diff
@@ -667,7 +674,8 @@ def get_containers(self, name, description=None):
             The port to connect to and a list of containers
         """
         data = {"endpoint_name": name, "description": description}
-        r = self._compute_web_client.v2.post("/v2/get_containers", data=data)
+        with self._request_lock:
+            r = self._compute_web_client.v2.post("/v2/get_containers", data=data)
```

Comment:
This return of r. is outside the with block, as opposed to line 721 below, should be consistent, no?  As with all of the results, then return r.data... in the file

Reply:
To put brief context, the short of it is that whether to return the `.data` from within the lock or after the lock, is syntactic sugar for the same thing.  In either case, the lock will be released _before_ the return value is given to the call site.